samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs

```diff
@@ -1,11 +1,11 @@
-﻿using Microsoft.ML.Data;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
+﻿#region ImageNetDataUsings
```

Comment:
Please remove these #region ....we need to keep the code consistent with other samples

Reply:
#resolved